src/pfs/pfs.proto

```diff
@@ -346,6 +346,7 @@ message AddFile {
   message URLSource {
     string URL = 1;
     bool recursive = 2;
+    int64 concurrency = 3;
```

Comment:
I was a little unclear on where this should go, but I think it makes sense here.  Any URLSource is something we could import using our new Import process.

Reply:
Can you make this an `uint32` or `uint64`, that way we don't have to worry about the negative case?

Reply:
Sure can do